Homework-05.ipynb

```diff
@@ -0,0 +1,197 @@
+      "{'items': [{'title': 'Confucius beyond the Analects', 'authors': 'Michael Hunter', 'itemType': 'Monograph'}, {'title': 'Dividing the realm in order to govern: the spatial organization of the Song state, 960-1276 CE', 'authors': 'Ruth Mostern', 'itemType': 'Monograph'}]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# your code\n",
+    "with open('HW-05-data.json', 'r') as f:\n",
+    "    output = json.load(f)\n",
+    "print(output)\n",
+    "\n",
+    "import json \n",
+    "dictionary ={'items':[\n",
+    "    {'title': 'Confucius beyond the Analects', \n",
+    "    'authors': 'Michael Hunter', \n",
+    "    'itemType': 'Monograph'\n",
+    "    },\n",
+    "    {'title': 'Dividing the realm in order to govern: the spatial organization of the Song state, 960-1276 CE',\n",
+    "     'authors': 'Ruth Mostern',\n",
+    "     'itemType': 'Monograph'\n",
+    "    }\n",
+    " ]\n",
+    "            }\n",
+    "json_object = json.dumps(dictionary, indent = 4) \n",
+    "with open(\"sample.json\", \"w\") as f: \n",
+    "    f.write(json_object) \n",
+    "with open('sample.json', 'r') as f:\n",
+    "    output = json.load(f)\n",
+    "print(output)"
```

Comment:
The assignment was "Then let the user enter new entries (consisting of at least a title, an author, and an item type)", but other than that good.